File: EkokBulma/EkokBulma.java
package ekokbulma;

import java.util.Scanner;

public class EkokBulma {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("Birinci sayıyı girin : ");
        int birinciSayi = reader.nextInt();
        System.out.print("İkinci sayıyı girin : ");
        int ikinciSayi = reader.nextInt();
        int ekok = Integer.max(birinciSayi, ikinciSayi);
        while (ekok <= birinciSayi * ikinciSayi) {
            if (ekok % birinciSayi == 0 && ekok % ikinciSayi == 0) {
                System.out.println(birinciSayi + " ve " + ikinciSayi + " sayılarının ebobu : " + ekok);
                return;
            } else {
                ekok++;
            }
        }
    }
}

File: EkokBulma/EkokBulma.py
def ekok_bul(birinci_sayi, ikinci_sayi):

    ekok = max(birinci_sayi, ikinci_sayi)

    while ekok <= birinci_sayi * ikinci_sayi:
        if (ekok % birinci_sayi == 0) and (ekok % ikinci_sayi == 0):
            print(ekok)
            return
        else:
            ekok += 1

ekok_bul(int(input("Bir sayi girin : ")), int(input("Bir sayi girin : ")))